fastapi_authorization_gateway/permissions.py

```diff
@@ -40,19 +40,24 @@ def params_match_permission(
     if permission_params is None:
         logger.debug("No params defined on policy. Match.")
         return True
-    else:
-        param_validator = generate_param_validator(permission_params)
-        try:
-            logger.debug(f"Request params: {request_params}")
-            param_validator(**request_params)
-        except ValidationError as err:
-            logger.debug(
-                "Params do not match permission constraints.", extra={"error": err}
-            )
-            return False
-        else:
-            logger.debug("Params do match permission constraints.")
-            return True
+    
+    if not request_params:
+        logger.debug("No request_params provided. No match.")
+        return False
```

Comment:
@edkeeble This is the only new logic added in this PR.

Reply:
Is this necessary? If an empty dict is passed in won't the param_validator catch it below?